### PR TITLE
Update gating/nightly tests to Fedora 29

### PR DESCRIPTION
**Pull request by dhnunes**
Testing new Fedora 29 Vagrant template box with gating tests.

### CODE REVIEW

**Comment by tiran** [comment]
`` RunPytest Failed to publish artifacts``, re-run.

**Comment by dhnunes** [comment]
I did a cleanup in Vagrant Cloud freeipa profile and changed the version to correct initial number ( 0.1.0 ) the box_uploader script has configured it initially to  0.0.11.

**Comment by netoarmando** [comment]
Please, update nightly definitions as well.

Also, as @f-trivino mentioned, improve your commit message.

**Comment by f-trivino** [comment]
> Please, update nightly definitions as well.
> 
> Also, as @f-trivino mentioned, improve your commit message.

I agree, gating is passing so I think we could add to this commit nightly changes too.

**Comment by tiran** [comment]
Are you planning to drop testing on F28 completely? I think that's a bad idea. F28 and F29 are going to diverge rather sooner than later. There should be some testing on F28.

File: ipatests/prci_definitions/gating.yaml [hunk 1]
```diff
@@ -13,232 +13,233 @@ topologies:
...
       class: Build
       args:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
-        template: &ci-master-f28
-          name: freeipa/ci-master-f28
-          version: 0.1.9
+        template: &ci-master-f29
+          name: freeipa/ci-master-f29
+          version: 0.1.0
```
Comment (netoarmando) [inline ipatests/prci_definitions/gating.yaml]: Update it to `0.1.1`, so 'firewalld' will be included.

File: ipatests/prci_definitions/nightly-f28.yaml [hunk 2]
```diff
@@ -0,0 +1,932 @@
...
+    requires: []
+    priority: 100
+    job:
+      class: Build
+      args:
+        git_repo: '{git_repo}'
+        git_refspec: '{git_refspec}'
+        template: &ci-master-f28
+          name: freeipa/ci-master-f28
+          version: 0.1.9
```
Comment (netoarmando) [inline ipatests/prci_definitions/nightly-f28.yaml]: Update it to `0.1.10`, so 'firewalld' will be included.

File: ipatests/prci_definitions/nightly_master.yaml [hunk 3]
```diff
@@ -25,908 +25,908 @@ topologies:
...
       class: Build
       args:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
-        template: &ci-master-f28
-          name: freeipa/ci-master-f28
-          version: 0.1.9
+        template: &ci-master-f29
+          name: freeipa/ci-master-f29
+          version: 0.1.0
```
Comment (netoarmando) [inline ipatests/prci_definitions/nightly_master.yaml]: Update it to `0.1.1`, so 'firewalld' will be included.

File: ipatests/prci_definitions/nightly-f28.yaml [hunk 4]
```diff
@@ -0,0 +1,932 @@
...
+    cpu: 2
+    memory: 4800
+  master_2repl_1client: &master_2repl_1client
+    name: master_2repl_1client
+    cpu: 5
+    memory: 9100
+  master_3repl_1client: &master_3repl_1client
+    name: master_3repl_1client
+    cpu: 6
+    memory: 11500
```
Comment (f-trivino) [inline ipatests/prci_definitions/nightly-f28.yaml]: 12900

**Comment by tiran** [comment]
The tests are failing because of authselect 1.0.2, see https://github.com/freeipa/freeipa/pull/2610

**Comment by f-trivino** [comment]
master:

* 25cfeea7691a90cb6cf5ee591504f572240dc084 PR-CI: Move to Fedora 29 template, version 0.2.0

**Pull request closed without merging**